tilecloud/scripts/tc_viewer.py

```diff
@@ -125,6 +127,8 @@ def index():
 
 
 def main() -> None:
+    global _root
```

Comment:
is this thread safe? What happens when 2 concurrent requests arrive?

Reply:
Yes, it's set only once before the server starts :-)

And I agree that not so nice...